### PR TITLE
Adding BlocklyAutomation

### PR DESCRIPTION
**Pull request by ignatandrei**

### CODE REVIEW

File: _data/tools.yml [hunk 1]
```diff
@@ -1885,3 +1885,17 @@
...
+  category: 
+    - code-generators
+    - sdk
+    - documentation
+    - testing
+  link: https://ignatandrei.github.io/BlocklyAutomation/
+  github: http://github.com/ignatandrei/BlocklyAutomation
+  language: Javascript / .NET
+  description:
+    Input any OPEN API definition file in order to have generated  Blocks in Blockly form to test and generate documentation
```
Comment (inverse) [inline _data/tools.yml:1899]:
```suggestion
    Input any OpenAPI definition file in order to have generated Blocks in Blockly form to test and generate documentation    
```

**Review by inverse — COMMENTED** [review]
Looks good aside this minor typo

**Comment by ignatandrei** [comment]
Approved now

File: _data/tools.yml [hunk 2]
```diff
@@ -1885,3 +1885,17 @@
...
+   
+- name: BlocklyAutomation
+  category: 
+    - code-generators
+    - sdk
+    - documentation
+    - testing
+  link: https://ignatandrei.github.io/BlocklyAutomation/
+  github: http://github.com/ignatandrei/BlocklyAutomation
+  language: Javascript / .NET
```
Comment (inverse) [inline _data/tools.yml:1897]:
```suggestion
  language: SaaS
```

Would SaaS fit this better as it's not something you run locally right?

Comment (ignatandrei) [reply _data/tools.yml:1897]: You can run this locally. There is a Microsoft Store app ( https://www.microsoft.com/en-us/p/blocklyautomation/9mxnwjt6zpd0 , will be updated monthly ) and a desktop application for Windows , updated more frequently - http://ba.serviciipeweb.ro/ . The desktop app has no CORS problems. So, no, it is not just  SAAS application

File: _data/tools.yml [hunk 3]
```diff
@@ -1885,3 +1885,17 @@
...
+  description:
+    Input any OpenAPI definition file in order to have generated Blocks in Blockly form to test and generate documentation
```
Comment (philsturgeon) [inline _data/tools.yml:1899]:
```suggestion
    Input any OpenAPI document to have generated Blocks in Blockly form to test and generate documentation.
```

Comment (philsturgeon) [reply _data/tools.yml:1899]: Also what is a block? This feels like we are defining something with the word, instead of explaining what it does.

**Comment by ignatandrei** [comment]
Could you please merge?

**Pull request merged**